ampersand-calendar.js

```diff
@@ -0,0 +1,290 @@
+;(function() {
+  var d3 = require('d3');
+  var _ = require('lodash');
+  var moment = require('moment');
+
+  var AmpersandState = require('ampersand-state');
+  var AmpersandView = require('ampersand-view');
+
+  var CalendarState = AmpersandState.extend({
+    session: {
+      startDate: [ 'object', false, function() { return null; } ],
```

Comment:
what do you think about making `startDate` and `endDate` part of the `props` rather than `session`?  That way `serialize`/`toJSON` return the interesting data?